src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -43,8 +47,15 @@ public void storeMessage(Message message) {
     messageEntity.setProperty("user", message.getUser());
     messageEntity.setProperty("text", message.getText());
     messageEntity.setProperty("timestamp", message.getTimestamp());
+    
+    int messageLength = message.getText().length();
```

Comment:
Since this doesn't have to deal with messageEntity, I'd say put it after the datastore.put so things are organized in logical blocks a bit better